aries_vcx/src/protocols/proof_presentation/verifier/states/finished.rs

```diff
@@ -1,15 +1,32 @@
-use crate::protocols::proof_presentation::verifier::state_machine::RevocationStatus;
+use serde::{Deserialize, Deserializer};
+
 use messages::concepts::problem_report::ProblemReport;
 use messages::protocols::proof_presentation::presentation::Presentation;
 use messages::protocols::proof_presentation::presentation_request::PresentationRequest;
 use messages::status::Status;
 
+use crate::protocols::proof_presentation::verifier::verification_status::PresentationVerificationStatus;
+
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct FinishedState {
     pub presentation_request: Option<PresentationRequest>,
     pub presentation: Option<Presentation>,
     pub status: Status,
-    pub revocation_status: Option<RevocationStatus>,
+    #[serde(
+        default = "PresentationVerificationStatus::Unavailable",
```

Comment:
I assume this is why you gave the `Unavailable` variant parenthesis? How about we rather implement `Default` for `PresentationVerificationStatus` and use 
```
#[serde(default, ...)]
```
here?

Reply:
correct, that's reason for parenthesis

I'd prefer to keep this explicit , I think `default` should be used where it really somewhat conceptually make sense to make certain value default. But in this case, it's only temporary migration/compatibility reason.

Reply:
I removed empty tuple parameter on `Unavailable` because it was causing issue with de/serialization - it ended up being serialized as 
```
{ ... "verification_status": { "Unavailable": []} }
```
and even failed to be deserialized - not sure but looked like serde bug.

So anyway, I changed it but exactly as you suggested, instead I refer to function `verification_status_unavailable` as default builder in this case. Since it's only local case where we need this default, I think it's not appropriate global default for the enum.

Reply:
I don't think it's a serde bug. Why would the unit type be present in the `Unavailable` variant?

Reply:
Well I would expect that if a structure is serialized, it would be successfully deserialized back. Perhaps it was related to the custom deserialization?
Anyway, it doesn't matter anymore.